Add docs for disabling usage tips in Granted (#228)
common-fate/docs: docs/granted/recipes/disable-usage-tips.md

# Disabling usage tips

> By default, Granted provides built-in tips to help users navigate the tool. However, some may find them distracting or unnecessary. With the latest release of Granted, you can disable this feature with an easy command:

Reviewer: Minor nitpicks here:

- "the latest release" will be confusing when we release new versions - it's easier to leave this out and assume that a user is going to be running the latest version.
- "an easy command" feels like we're trying to sell to the user, it's better to take an informative rather than sales-ey tone in the docs

```suggestion
By default, Granted provides built-in tips to help users navigate the tool. However, some may find them distracting or unnecessary. You can disable this feature by running the following command:
```
